packages/opentelemetry-metrics/src/BoundInstrument.ts

```diff
@@ -53,18 +71,16 @@ export class BoundCounter extends BaseBoundInstrument
   implements types.BoundCounter {
   constructor(
     labelSet: types.LabelSet,
-    private readonly _disabled: boolean,
-    private readonly _monotonic: boolean,
-    private readonly _valueType: types.ValueType,
-    private readonly _logger: types.Logger,
-    private readonly _onUpdate: Function
+    disabled: boolean,
+    monotonic: boolean,
```

Comment:
Shouldn't a counter always be monotonic?

Reply:
Counters are defined as `monotonic=true` by default (positive values only), but it is possible to set `monotonic=false` (as an option) in which case they support +/- increments (IMO it is then same as `Gauge`.)